services/alertmanager/alertmanager.go

```diff
@@ -78,7 +82,15 @@ func (svc *Service) Run(ctx context.Context) {
 // TODO That's a temporary measure until we start generating /etc/alertmanager.yml
 // using /srv/alertmanager/alertmanager.base.yml as a base. See supervisord config.
 func (svc *Service) generateBaseConfig() {
-	const path = "/srv/alertmanager/alertmanager.base.yml"
+	// create alert manager directories if not exists in the persistent volume.
+	if _, err := os.Stat(alertmanagerDataDir); os.IsNotExist(err) {
+		svc.l.Infof("Creating %q", alertmanagerDataDir)
+		if err := os.MkdirAll(alertmanagerDataDir, os.ModePerm); err != nil {
+			svc.l.Errorf("Cannot create datadir for AlertManager %v.", err)
+			os.Exit(1)
```

Comment:
Absolutely not. It is too late for that there. The only thing we could do is to log a error and return.